core/src/stores/store.rs

```diff
@@ -229,6 +230,8 @@ pub trait Store {
         &self,
         project: &Project,
         data_source_id: &str,
```

Comment:
order of arguments should be coherent between these two functions ideally

Reply:
Yep let me do it.